collect_app/src/main/java/org/odk/collect/android/widgets/StringWidget.java

```diff
@@ -45,7 +55,7 @@ protected StringWidget(Context context, QuestionDetails questionDetails, boolean
 
     protected void setUpLayout() {
         setDisplayValueFromModel();
-        addAnswerView(answerText);
+        addAnswerView(answerText, 12);
```

Comment:
Seems `12` should be a named value?

Reply:
Yeah it's just temporary value - we'll hopefully rework the layout so we don't have old school weird padding on the `EditText` (we're making it look like `16dp` padding). But yeah naming it/commenting it would have been a nice thing to do that I will do now 😇